README.md

```diff
@@ -1,56 +1,119 @@
-[Racket](https://racket-lang.org/) is a general-purpose programming
-language and an ecosystem for language-oriented programming.
+# Racket Programming Language
 
-This repository holds the source code for the core of Racket plus some
-related packages. The rest of the Racket distribution source code is
-in other repositories, mostly under [the Racket GitHub
-organization](https://github.com/racket).
+![License: MIT/Apache/LGPL](https://img.shields.io/badge/license-MIT%2FApache%2FLGPL-blue)
+![Made with Scheme](https://img.shields.io/badge/Made%20with-Scheme-lightgrey?logo=racket)
 
-Quick Start
------------
+<img width="396" height="120" alt="image" src="https://github.com/user-attachments/assets/0526b430-699e-488d-9378-7040f8458c72" />
```

Comment:
Not sure if it makes sense or whether it's possible or not, but would it be better to add the logo `.svg` file to the Racket repo and embed it instead? The image is blurry.
https://racket-lang.org/img/racket-logo.svg

Reply:
(@mflatt: is adding `racket-logo.svg` to `racket/racket` a sensible thing to do?)